### ISSUE TITLE
Positive Tabindex Values

### ISSUE THREAD

**Issue by isralduke** [la-ots/pelican#402]
Two items Topbar include have positive tabindex values. This is considered bad practice for accessibility. They exist because they were a part of the third-party code used in the early versions of Pelican. The markup should be refactored so that the tabindex values aren’t necessary.

File: `_includes/app-screen.njk`
